src/style.css

```diff
@@ -0,0 +1,76 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(.\fonts\Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(.\fonts\Avenir-Heavy.ttf);
+  font-weight: 700;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.input {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  height: 70px;
+  padding-left: 62px;
+  width: 100%;
+}
+
+.formButton {
+  border: none;
+  display: block;
+  width: 19px;
+  height: 19px;
+  background: url(./images/Search.svg);
+  position: absolute;
+  left: 26px;
+  top: 25px;
+}
+
+.input::placeholder {
+  font-weight: 300;
+  color: #3d4e61;
+  text-shadow: none;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: bold;
+  outline: none;
+}
+
+.form[data-qa="small"] > .input {
+  height: 42px;
+  font-size: 14px;
+  padding-left: 33px;
+}
+
+.form[data-qa="small"] .formButton {
```

Comment:
don't increase the specificity. use a special modifier class for this: `.input--small`, `.formButton--small`